Framework/src/TaskDPL.cxx

```diff
@@ -0,0 +1,95 @@
+///
+/// \file    TaskDPL.cxx
+/// \author  Piotr Konopka
+///
+/// \brief This is an executable showing QC Task's usage in Data Processing Layer.
+///
+/// This is an executable showing QC Task's usage in Data Processing Layer. The workflow consists of data producer,
+/// which generates arrays of random size and content. Its output is dispatched to QC task using Data Sampling
+/// infrastructure. QC Task runs exemplary user code located in SkeletonDPL. The resulting historgram contents
+/// are printed by checker.
+/// QC task is instantiated by TaskDataProcessorFactory with preinstalled config file, which can be found in
+/// ${QUALITYCONTROL_ROOT}/etc/qcTaskDplConfig.ini or Framework/qcTaskDplConfig.ini (original one).
+
```

Comment:
Could you maybe say a word about how to launch it and what to expect ? 
I launch it and it nicely starts and stops quickly and I don't know if it is normal.

Reply:
Also, what data is it getting and from where ?